### PR TITLE
Feature/prismjs dhall

### PR DESCRIPTION
**Pull request by JohannesRudolph**
This PR makes us use proper dhall syntax highlighting which improves the rendering of code markup. This is important as we use more dhall in the documentation and the language deviates further from haskell, which was the bandaid we used so far. 

It achieves this by swtiching our main syntax highlighting from highlight.js to prims.js. 

Uses the "tomorrow-night-eighties" theme because that's the closest one I could find to the existing prism.js theme that was working in docusaurus without a lot of adaptations (only background color & font size had to change).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,5 +35,5 @@ The following formatting should be applied for contribution to the meshcloud doc
 * For `buttons` use backticks (\`button\`).
 * No quotation marks around item names (") please.
 * Use the style as recommended by [markdownlint](https://github.com/markdownlint/markdownlint). For VSCode use the extensions `davidanson.vscode-markdownlint` to get inline linting.
-* Format `dhall` code in fenced code blocks using `haskell`. The `highlight.js`/`prism.js` used by docusaurus unfortunately don't support dhall natively yet.
+* Format `dhall` code in fenced code blocks using `dhall`. The `highlight.js`/`prism.js` used by docusaurus unfortunately don't support dhall natively yet.
```
Comment (JohannesRudolph) [inline CONTRIBUTING.md:38]: f: this is wrong - prism.js supports it natively

**Comment by SimonTheLeg** [comment]
for the curious: this is what it looks like now :) 

image

**Comment by SimonTheLeg** [comment]
I seem to be getting some "language does not exist dhall" errors, when navigating to a page with syntax highlighting

example http://localhost:3000/docs/meshstack.azure.index.html

```
Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`
LiveReload server started on port 35729
Docusaurus server started on port 3000
Language does not exist dhall
TypeError: Cannot read property 'rest' of undefined
    at Object.tokenize (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/prismjs/prism.js:414:22)
    at Object.highlight (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/prismjs/prism.js:280:18)
    at Function.highlight (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/docusaurus/lib/core/renderMarkdown.js:56:32)
    at Object.rules.fence (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/remarkable/lib/rules.js:73:37)
    at Renderer.render (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/remarkable/lib/renderer.js:71:39)
    at Remarkable.render (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/remarkable/lib/index.js:152:24)
    at MarkdownRenderer.toHtml (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/docusaurus/lib/core/renderMarkdown.js:114:26)
    at renderMarkdown (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/docusaurus/lib/core/renderMarkdown.js:124:43)
    at Remarkable.content (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/docusaurus/lib/core/Remarkable.js:17:21)
    at Remarkable.render (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/docusaurus/lib/core/Remarkable.js:36:29)
Language does not exist dhall
```

**Comment by SimonTheLeg** [comment]
Ah forget it, I've seen now you've split dhall off

**Comment by SimonTheLeg** [comment]
Also syntax highlighting does not seem to work for me

image

**Comment by JohannesRudolph** [comment]
> I seem to be getting some "language does not exist dhall" errors, when navigating to a page with syntax highlighting
> 
> example http://localhost:3000/docs/meshstack.azure.index.html
> 
> ```
> Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`
> LiveReload server started on port 35729
> Docusaurus server started on port 3000
> Language does not exist dhall
> TypeError: Cannot read property 'rest' of undefined
>     at Object.tokenize (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/prismjs/prism.js:414:22)
>     at Object.highlight (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/prismjs/prism.js:280:18)
>     at Function.highlight (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/docusaurus/lib/core/renderMarkdown.js:56:32)
>     at Object.rules.fence (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/remarkable/lib/rules.js:73:37)
>     at Renderer.render (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/remarkable/lib/renderer.js:71:39)
>     at Remarkable.render (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/remarkable/lib/index.js:152:24)
>     at MarkdownRenderer.toHtml (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/docusaurus/lib/core/renderMarkdown.js:114:26)
>     at renderMarkdown (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/docusaurus/lib/core/renderMarkdown.js:124:43)
>     at Remarkable.content (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/docusaurus/lib/core/Remarkable.js:17:21)
>     at Remarkable.render (/Users/simonbein/meshcloud/meshcloud-docs/website/node_modules/docusaurus/lib/core/Remarkable.js:36:29)
> Language does not exist dhall
> ```

Did you run a `yarn install` ? You'll need to install a prism.js update that's defined in `package.json`

**Comment by JohannesRudolph** [comment]
> Also syntax highlighting does not seem to work for me
After `yarn install` you should see 
image

**Comment by SimonTheLeg** [comment]
ah classis JavaScript... 🙈 Works now ✅

**Pull request merged**